rj_param_utils/CMakeLists.txt

```diff
@@ -41,6 +43,9 @@ if(BUILD_TESTING)
   target_link_libraries(rj_param_utils_testing PUBLIC rj_param_utils ${GTEST_LIBRARIES})
 endif()
 
+# position-independent-code flag
+target_compile_options(rj_param_utils PRIVATE "-fPIC")
```

Comment:
Can you check again to see if this is really necessary? I'm kind of confused as to why it wasn't working without it...